internal/compiler/tests/syntax/callbacks/init.slint

```diff
@@ -0,0 +1,17 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+Label := Text {
+
+}
+
+Test1 := Rectangle {
+    Rectangle {
+        init => { }
+//      ^error{'init' is not a callback in Rectangle}
```

Comment:
The error message should specify that it is only for top level component or in `for` and `if`